src/main/java/net/wasdev/wlp/common/plugins/util/InstallFeatureUtil.java

```diff
@@ -709,55 +711,49 @@ private void productInfoValidate() throws PluginExecutionException {
      * Runs the productInfo command and returns the output
      * 
      * @param installDirectory The directory of the installed runtime
-     * @param action The action to perform for the productInfo command
+     * @param action           The action to perform for the productInfo command
      * @return The command output
      * @throws PluginExecutionException if the exit value of the command was not 0
      */
-    public static String productInfo(File installDirectory, String action) throws PluginExecutionException {
```

Comment:
Use `public static` since this method is called from ci.maven's integration tests.